libs/langchain-google-common/src/connection.ts

```diff
@@ -245,6 +288,19 @@ export abstract class GoogleAIConnection<
     }
   }
 
+  get computedLocation(): string {
+    switch (this.apiName) {
+      case "google":
+        return super.computedLocation;
+      case "anthropic":
+        return "us-east5";
```

Comment:
Should we be hardcoding this here?

If it's only available in one region now, would prefer to have this configurable or even not have a default at all and just have it documented

Reply:
See above about making magic values as unnecessary as possible.

Some of the Anthropic models are available in other regions, but all are available in us-east5. And none, mysteriously, are available in us-central1.

I'd like to keep it as a default, but clearly have it available to be settable.